src/frontend/src/pages/LandingPage/LandingPage.scss

```diff
@@ -2,41 +2,33 @@
   display: flex;
   margin-top: calc($header-height);
   min-height: calc(100vh - $header-height);
-  @include breakpoint(0 map-get($grid-breakpoints, 'xl')) {
+  @include breakpoint(0 map-get($grid-breakpoints, 'lg')) {
      flex-direction: column;
      justify-content: center;
      align-items: center;
     }
 
-  @include breakpoint(map-get($grid-breakpoints, 'xl')) {
+  @include breakpoint(map-get($grid-breakpoints, 'lg')) {
     background-repeat: no-repeat;
     background-image: url(/src/assets/public_homepage/hero.svg);
     background-position: 90% 100%;
   }
 }
 
-.mobile-show-hero {
-  display: none;
-   @include breakpoint (0 map-get($grid-breakpoints, 'xl'))
-   { display: block; }
-  }
-
-
 .hero-content {
   display: flex;
   justify-content: center;
   flex-direction: column;
   max-width:45ch;
   
-  @include breakpoint(map-get($grid-breakpoints, 'xxl')) {
+  @include breakpoint(map-get($grid-breakpoints, 'md')) {
```

Comment:
should this be lg?

Reply:
if set to lg will become like this
<img width="1189" alt="MicrosoftTeams-image" src="https://user-images.githubusercontent.com/57013495/227274669-f3433875-4f35-4a1f-8fd4-ee7660e433fd.png">